test/PayID/pay-id-utils.test.ts

```diff
@@ -11,7 +11,7 @@ describe('PayIDUtils', function (): void {
     const rawPayID = `${path}$${host}`
 
     // WHEN it is parsed to components.
-    const payIDComponents = PayIDUtils.parsePayID(rawPayID)
+    const payIDComponents = PayIdUtils.parsePayID(rawPayID)
```

Comment:
does it matter that this `payIDComponents` variable is cased in the old style?  I know the style applies mostly to classnames but wondering if we would benefit from that consistency wherever possible?

Reply:
(here and throughout the file)